Lib/typing.py

```diff
@@ -497,6 +497,13 @@ def __getitem__(self, parameters):
         return self._getitem(self, *parameters)
 
 
+class _AnnotatedSpecialForm(_SpecialForm, _root=True):
```

Comment:
Or we can just name it `_TypedCacheSpecialForm` and use for both `Literal` and `Annotated`

Reply:
I like that idea. It's a private implementation detail, so hopefully nobody is relying on `typing._LiteralSpecialForm` existing (I can't find any uses on `grep.app`).